ocp_install_env.sh

```diff
@@ -47,10 +45,8 @@ function extract_rhcos_json() {
     pullsecret_file=$(mktemp --tmpdir "pullsecret--XXXXXXXXXX")
     _tmpfiles="$_tmpfiles $pullsecret_file"
```

Comment:
I think we can remove this tmpfile as it's no longer used?

Reply:
Done